From pearl-miui-camera-6.6-compat
Subject: [PATCH] Remove front/back camera switch in portrait mode

The camera switch key should not appear in portrait mode. When the
camera picker container is configured, hide it for portrait mode and
restore it for other modes.

---
 smali/z4/G.smali | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/smali/z4/G.smali b/smali/z4/G.smali
index 8f55247..7e12160 100644
--- a/smali/z4/G.smali
+++ b/smali/z4/G.smali
@@ -261,6 +261,32 @@
 
     iget-object v4, v0, Lz4/G;->a:Landroid/view/ViewGroup;
 
+    invoke-virtual {v4}, Landroid/view/View;->getId()I
+
+    move-result v3
+
+    const v5, 0x7f0b0bbc
+
+    if-ne v3, v5, :cond_1
+
+    const/16 v5, 0xab
+
+    if-ne v2, v5, :cond_0
+
+    const/16 v5, 0x8
+
+    invoke-virtual {v4, v5}, Landroid/view/View;->setVisibility(I)V
+
+    return-void
+
+    :cond_0
+    const/4 v5, 0x0
+
+    invoke-virtual {v4, v5}, Landroid/view/View;->setVisibility(I)V
+
+    :cond_1
+    iget-object v4, v0, Lz4/G;->a:Landroid/view/ViewGroup;
+
     const v5, 0x7f0b0840
 
     invoke-virtual {v4, v5}, Landroid/view/View;->getTag(I)Ljava/lang/Object;
@@ -273,7 +299,7 @@
 
     const/4 v8, 0x0
 
-    if-eqz v4, :cond_1
+    if-eqz v4, :cond_3
 
     move-object v9, v4
 
@@ -283,21 +309,21 @@
 
     move-result v9
 
-    if-ne v9, v1, :cond_1
+    if-ne v9, v1, :cond_3
 
-    if-nez p4, :cond_1
+    if-nez p4, :cond_3
 
-    if-eq v1, v6, :cond_1
+    if-eq v1, v6, :cond_3
 
     sget-object v9, LF1/D2;->f:LF1/D2;
 
     iget-boolean v9, v9, LF1/D2;->d:Z
 
-    if-nez v9, :cond_0
+    if-nez v9, :cond_2
 
     const/16 v9, 0xcc
 
-    if-eq v1, v9, :cond_0
+    if-eq v1, v9, :cond_2
 
     new-array v0, v8, [Ljava/lang/Object;
 
@@ -309,14 +335,14 @@
 
     return-void
 
-    :cond_0
+    :cond_2
     move-object v9, v7
 
     move v10, v8
 
     goto :goto_0
 
-    :cond_1
+    :cond_3
     move-object/from16 v9, p1
 
     move/from16 v10, p2
@@ -340,35 +366,35 @@
 
     move-result-object v12
 
-    if-eqz v12, :cond_2
+    if-eqz v12, :cond_4
 
     const/4 v12, 0x1
 
     goto :goto_1
 
-    :cond_2
+    :cond_4
     move v12, v8
 
     :goto_1
     iget-object v13, v0, Lz4/G;->k:Landroid/view/ViewGroup;
 
-    if-eqz v13, :cond_3
+    if-eqz v13, :cond_5
 
     invoke-virtual {v13}, Landroid/view/View;->getVisibility()I
 
     move-result v13
 
-    if-nez v13, :cond_3
+    if-nez v13, :cond_5
 
     const/4 v13, 0x1
 
     goto :goto_2
 
-    :cond_3
+    :cond_5
     move v13, v8
 
     :goto_2
-    if-eqz v4, :cond_4
+    if-eqz v4, :cond_6
 
     move-object v14, v4
 
@@ -380,9 +406,9 @@
 
     const/16 v15, 0xc4
 
-    if-ne v14, v15, :cond_4
+    if-ne v14, v15, :cond_6
 
-    if-eqz v12, :cond_4
+    if-eqz v12, :cond_6
 
     iget-object v14, v0, Lz4/G;->a:Landroid/view/ViewGroup;
 
@@ -392,7 +418,7 @@
 
     iput-object v14, v0, Lz4/G;->o:Landroid/graphics/drawable/Drawable;
 
-    :cond_4
+    :cond_6
     invoke-static {v1, v2}, Lz4/G;->a(II)Z
 
     move-result v14
@@ -403,26 +429,26 @@
 
     const/16 v15, 0x8
 
-    if-eqz v14, :cond_5
+    if-eqz v14, :cond_7
 
     invoke-virtual {v14, v15}, Landroid/view/View;->setVisibility(I)V
 
-    :cond_5
+    :cond_7
     iget-object v14, v0, Lz4/G;->n:Lz4/H;
 
-    if-eqz v14, :cond_6
+    if-eqz v14, :cond_8
 
     invoke-virtual {v14}, Landroid/view/View;->getVisibility()I
 
     move-result v14
 
-    if-eq v14, v15, :cond_6
+    if-eq v14, v15, :cond_8
 
     iget-object v14, v0, Lz4/G;->n:Lz4/H;
 
     invoke-virtual {v14, v15}, Landroid/view/View;->setVisibility(I)V
 
-    :cond_6
+    :cond_8
     iget-object v14, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
     invoke-virtual {v14, v7}, Landroid/view/View;->setBackground(Landroid/graphics/drawable/Drawable;)V
@@ -435,7 +461,7 @@
 
     const/high16 v15, 0x3f800000    # 1.0f
 
-    if-eq v1, v14, :cond_19
+    if-eq v1, v14, :cond_1b
 
     const v14, 0x7f060b97
 
@@ -474,7 +500,7 @@
 
     move-object/from16 v16, v4
 
-    :cond_7
+    :cond_9
     :goto_3
     const/4 v6, 0x1
 
@@ -513,7 +539,7 @@
 
     check-cast v11, Lr2/B;
 
-    if-eqz v11, :cond_9
+    if-eqz v11, :cond_b
 
     const/16 v14, 0x100
 
@@ -535,13 +561,13 @@
 
     move-result v11
 
-    if-eqz v11, :cond_8
+    if-eqz v11, :cond_a
 
     const v11, 0x7f080a81
 
     goto :goto_4
 
-    :cond_8
+    :cond_a
     const v11, 0x7f080a83
 
     :goto_4
@@ -551,7 +577,7 @@
 
     invoke-virtual {v6, v2}, Landroid/view/View;->setContentDescription(Ljava/lang/CharSequence;)V
 
-    :cond_9
+    :cond_b
     iget-object v6, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
     new-instance v11, Lz4/G$b;
@@ -591,7 +617,7 @@
 
     iget-object v4, v0, Lz4/G;->n:Lz4/H;
 
-    if-nez v4, :cond_a
+    if-nez v4, :cond_c
 
     new-instance v4, Lz4/H;
 
@@ -637,13 +663,13 @@
 
     goto :goto_5
 
-    :cond_a
+    :cond_c
     const/4 v6, 0x1
 
     :goto_5
     iget-object v2, v0, Lz4/G;->l:Landroid/graphics/Paint;
 
-    if-nez v2, :cond_b
+    if-nez v2, :cond_d
 
     new-instance v2, Landroid/graphics/Paint;
 
@@ -681,10 +707,10 @@
 
     invoke-virtual {v2, v4}, Landroid/graphics/Paint;->setColor(I)V
 
-    :cond_b
+    :cond_d
     iget-object v2, v0, Lz4/G;->m:Landroid/graphics/Paint;
 
-    if-nez v2, :cond_c
+    if-nez v2, :cond_e
 
     new-instance v2, Landroid/graphics/Paint;
 
@@ -702,7 +728,7 @@
 
     invoke-virtual {v2, v4}, Landroid/graphics/Paint;->setStyle(Landroid/graphics/Paint$Style;)V
 
-    :cond_c
+    :cond_e
     sget-object v2, Lo9/a;->a:Lo9/b;
 
     invoke-interface {v2}, Lo9/b;->g()Lp9/c;
@@ -733,13 +759,13 @@
 
     move-result v3
 
-    if-eqz v3, :cond_d
+    if-eqz v3, :cond_f
 
     iget-object v3, v0, Lz4/G;->n:Lz4/H;
 
     invoke-virtual {v3, v8}, Landroid/view/View;->setVisibility(I)V
 
-    :cond_d
+    :cond_f
     iget-object v3, v0, Lz4/G;->n:Lz4/H;
 
     new-instance v4, Ljava/lang/StringBuilder;
@@ -850,7 +876,7 @@
 
     invoke-virtual {v3, v4}, Lcom/airbnb/lottie/LottieAnimationView;->setImageResource(I)V
 
-    if-eqz v11, :cond_7
+    if-eqz v11, :cond_9
 
     sget-object v3, Lf2/e;->c:Lf2/e;
 
@@ -964,7 +990,7 @@
 
     move-result-object v3
 
-    if-nez v2, :cond_f
+    if-nez v2, :cond_11
 
     const-string v2, "0"
 
@@ -972,16 +998,16 @@
 
     move-result v2
 
-    if-eqz v2, :cond_e
+    if-eqz v2, :cond_10
 
     goto :goto_7
 
-    :cond_e
+    :cond_10
     move v2, v8
 
     goto :goto_8
 
-    :cond_f
+    :cond_11
     :goto_7
     const/4 v2, 0x1
 
@@ -992,13 +1018,13 @@
 
     move-result-object v4
 
-    if-eqz v2, :cond_10
+    if-eqz v2, :cond_12
 
     const v6, 0x7f14100a
 
     goto :goto_9
 
-    :cond_10
+    :cond_12
     const v6, 0x7f14100c
 
     :goto_9
@@ -1170,7 +1196,7 @@
 
     invoke-virtual {v3, v4}, Lcom/airbnb/lottie/LottieAnimationView;->setImageResource(I)V
 
-    if-eqz v11, :cond_7
+    if-eqz v11, :cond_9
 
     iget-object v3, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
@@ -1238,7 +1264,7 @@
 
     invoke-virtual {v3, v4}, Lcom/airbnb/lottie/LottieAnimationView;->setImageResource(I)V
 
-    if-eqz v11, :cond_7
+    if-eqz v11, :cond_9
 
     iget-object v3, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
@@ -1271,7 +1297,7 @@
 
     iget-boolean v3, v0, Lz4/G;->c:Z
 
-    if-eqz v3, :cond_11
+    if-eqz v3, :cond_13
 
     iget-object v3, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
@@ -1291,7 +1317,7 @@
 
     goto :goto_a
 
-    :cond_11
+    :cond_13
     iget-object v3, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
     sget-object v4, Lo9/a;->a:Lo9/b;
@@ -1311,7 +1337,7 @@
     :goto_a
     iget-object v3, v0, Lz4/G;->o:Landroid/graphics/drawable/Drawable;
 
-    if-eqz v3, :cond_12
+    if-eqz v3, :cond_14
 
     iget-object v4, v0, Lz4/G;->a:Landroid/view/ViewGroup;
 
@@ -1319,7 +1345,7 @@
 
     goto :goto_b
 
-    :cond_12
+    :cond_14
     iget-object v3, v0, Lz4/G;->a:Landroid/view/ViewGroup;
 
     invoke-virtual {v3, v7}, Landroid/view/View;->setBackground(Landroid/graphics/drawable/Drawable;)V
@@ -1338,7 +1364,7 @@
 
     invoke-virtual {v2}, Landroid/widget/ImageView;->clearColorFilter()V
 
-    if-eqz v16, :cond_13
+    if-eqz v16, :cond_15
 
     move-object/from16 v4, v16
 
@@ -1350,13 +1376,13 @@
 
     const/16 v3, 0xc2
 
-    if-ne v2, v3, :cond_13
+    if-ne v2, v3, :cond_15
 
     move-object v9, v7
 
     move v10, v8
 
-    :cond_13
+    :cond_15
     iget-object v2, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
     invoke-virtual {v2}, Landroid/view/View;->getContext()Landroid/content/Context;
@@ -1369,7 +1395,7 @@
 
     move-result-object v2
 
-    if-eqz v11, :cond_14
+    if-eqz v11, :cond_16
 
     sget-object v3, Lf2/e;->c:Lf2/e;
 
@@ -1387,7 +1413,7 @@
 
     invoke-virtual {v4, v3}, Landroid/widget/ImageView;->setColorFilter(Landroid/graphics/ColorFilter;)V
 
-    :cond_14
+    :cond_16
     sget-object v3, Lo9/a;->a:Lo9/b;
 
     invoke-interface {v3}, Lo9/b;->g()Lp9/c;
@@ -1413,7 +1439,7 @@
 
     invoke-virtual {v2}, Landroid/widget/ImageView;->clearColorFilter()V
 
-    if-eqz v16, :cond_15
+    if-eqz v16, :cond_17
 
     move-object/from16 v4, v16
 
@@ -1425,13 +1451,13 @@
 
     const/16 v3, 0xc3
 
-    if-ne v2, v3, :cond_15
+    if-ne v2, v3, :cond_17
 
     move-object v9, v7
 
     move v10, v8
 
-    :cond_15
+    :cond_17
     iget-object v2, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
     invoke-virtual {v2}, Landroid/view/View;->getContext()Landroid/content/Context;
@@ -1444,7 +1470,7 @@
 
     move-result-object v2
 
-    if-eqz v11, :cond_16
+    if-eqz v11, :cond_18
 
     sget-object v3, Lf2/e;->c:Lf2/e;
 
@@ -1462,7 +1488,7 @@
 
     invoke-virtual {v4, v3}, Landroid/widget/ImageView;->setColorFilter(Landroid/graphics/ColorFilter;)V
 
-    :cond_16
+    :cond_18
     sget-object v3, Lo9/a;->a:Lo9/b;
 
     invoke-interface {v3}, Lo9/b;->g()Lp9/c;
@@ -1488,7 +1514,7 @@
 
     iget-boolean v2, v2, LF1/D2;->d:Z
 
-    if-nez v2, :cond_17
+    if-nez v2, :cond_19
 
     iget-object v2, v0, Lz4/G;->a:Landroid/view/ViewGroup;
 
@@ -1498,13 +1524,13 @@
 
     const v3, 0x7f0b0cd1
 
-    if-eq v2, v3, :cond_17
+    if-eq v2, v3, :cond_19
 
     iget-object v2, v0, Lz4/G;->a:Landroid/view/ViewGroup;
 
     invoke-virtual {v2, v7}, Landroid/view/View;->setOnClickListener(Landroid/view/View$OnClickListener;)V
 
-    :cond_17
+    :cond_19
     iget-object v2, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
     invoke-virtual {v2}, Landroid/widget/ImageView;->clearColorFilter()V
@@ -1545,13 +1571,13 @@
 
     move-result v4
 
-    if-eqz v4, :cond_18
+    if-eqz v4, :cond_1a
 
     const v4, 0x7f140cd4
 
     goto :goto_c
 
-    :cond_18
+    :cond_1a
     const v4, 0x7f140cd2
 
     :goto_c
@@ -1587,7 +1613,7 @@
 
     invoke-virtual {v4, v3}, Lcom/airbnb/lottie/LottieAnimationView;->setImageDrawable(Landroid/graphics/drawable/Drawable;)V
 
-    if-eqz v11, :cond_7
+    if-eqz v11, :cond_9
 
     sget-object v3, Lf2/e;->c:Lf2/e;
 
@@ -1620,7 +1646,7 @@
 
     goto :goto_d
 
-    :cond_19
+    :cond_1b
     move-object/from16 v16, v4
 
     iget-object v2, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
@@ -1658,7 +1684,7 @@
 
     invoke-virtual {v3, v5, v4}, Landroid/view/View;->setTag(ILjava/lang/Object;)V
 
-    if-eqz v16, :cond_1a
+    if-eqz v16, :cond_1c
 
     move-object/from16 v4, v16
 
@@ -1670,17 +1696,17 @@
 
     const/16 v4, 0xc9
 
-    if-ne v3, v4, :cond_1a
+    if-ne v3, v4, :cond_1c
 
     const/4 v3, 0x1
 
     goto :goto_e
 
-    :cond_1a
+    :cond_1c
     move v3, v8
 
     :goto_e
-    if-eqz v16, :cond_1b
+    if-eqz v16, :cond_1d
 
     move-object/from16 v4, v16
 
@@ -1690,20 +1716,20 @@
 
     move-result v4
 
-    if-ne v4, v1, :cond_1b
+    if-ne v4, v1, :cond_1d
 
     move v10, v8
 
-    :cond_1b
+    :cond_1d
     iget-object v4, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
     iget-boolean v5, v0, Lz4/G;->h:Z
 
     const/16 v11, 0xd3
 
-    if-nez v5, :cond_1c
+    if-nez v5, :cond_1e
 
-    if-eq v1, v11, :cond_1c
+    if-eq v1, v11, :cond_1e
 
     iget-object v5, v0, Lz4/G;->a:Landroid/view/ViewGroup;
 
@@ -1711,12 +1737,12 @@
 
     goto :goto_f
 
-    :cond_1c
-    if-eqz v5, :cond_1d
+    :cond_1e
+    if-eqz v5, :cond_1f
 
-    if-nez v12, :cond_1d
+    if-nez v12, :cond_1f
 
-    if-eq v1, v11, :cond_1d
+    if-eq v1, v11, :cond_1f
 
     invoke-static {}, Lg2/a;->j()Lv2/C0;
 
@@ -1730,11 +1756,11 @@
 
     check-cast v5, Lv2/E0;
 
-    if-eqz v5, :cond_1d
+    if-eqz v5, :cond_1f
 
     iget-object v5, v5, Lv2/E0;->b:Lv2/F0;
 
-    if-eqz v5, :cond_1d
+    if-eqz v5, :cond_1f
 
     iget v5, v5, Lv2/F0;->e:I
 
@@ -1752,26 +1778,26 @@
 
     invoke-static {v5, v14}, Lcom/android/camera/fragment/m;->f(Z[Landroid/view/View;)V
 
-    :cond_1d
+    :cond_1f
     :goto_f
     iget-object v5, v0, Lz4/G;->k:Landroid/view/ViewGroup;
 
-    if-eqz v5, :cond_1e
+    if-eqz v5, :cond_20
 
     invoke-virtual {v5}, Landroid/view/View;->getVisibility()I
 
     move-result v5
 
-    if-nez v5, :cond_1e
+    if-nez v5, :cond_20
 
     const/4 v8, 0x1
 
-    :cond_1e
+    :cond_20
     const/16 v5, 0xc0
 
-    if-eq v1, v5, :cond_20
+    if-eq v1, v5, :cond_22
 
-    if-eqz v16, :cond_1f
+    if-eqz v16, :cond_21
 
     move-object/from16 v7, v16
 
@@ -1781,38 +1807,38 @@
 
     move-result v7
 
-    if-eq v7, v5, :cond_20
+    if-eq v7, v5, :cond_22
 
-    :cond_1f
+    :cond_21
     iget-boolean v5, v0, Lz4/G;->h:Z
 
-    if-ne v12, v5, :cond_20
+    if-ne v12, v5, :cond_22
 
-    if-eq v13, v8, :cond_21
+    if-eq v13, v8, :cond_23
 
-    :cond_20
+    :cond_22
     iget-object v4, v0, Lz4/G;->a:Landroid/view/ViewGroup;
 
-    :cond_21
+    :cond_23
     const/high16 v5, 0x3f000000    # 0.5f
 
-    if-eqz v6, :cond_2a
+    if-eqz v6, :cond_2c
 
-    if-eqz v3, :cond_22
+    if-eqz v3, :cond_24
 
     goto :goto_10
 
-    :cond_22
+    :cond_24
     const/4 v5, 0x0
 
     :goto_10
     iget-object v3, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
-    if-ne v4, v3, :cond_24
+    if-ne v4, v3, :cond_26
 
     iget-boolean v3, v0, Lz4/G;->f:Z
 
-    if-eqz v3, :cond_23
+    if-eqz v3, :cond_25
 
     iget-object v3, v0, Lz4/G;->a:Landroid/view/ViewGroup;
 
@@ -1820,16 +1846,16 @@
 
     goto :goto_11
 
-    :cond_23
+    :cond_25
     iget-object v3, v0, Lz4/G;->a:Landroid/view/ViewGroup;
 
     invoke-static {v3}, LU1/b;->e(Landroid/view/View;)V
 
-    :cond_24
+    :cond_26
     :goto_11
     const/16 v3, 0xd2
 
-    if-ne v1, v3, :cond_25
+    if-ne v1, v3, :cond_27
 
     iget-object v1, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
@@ -1839,7 +1865,7 @@
 
     goto :goto_12
 
-    :cond_25
+    :cond_27
     iget-object v1, v0, Lz4/G;->b:Lcom/airbnb/lottie/LottieAnimationView;
 
     invoke-virtual {v1, v2}, Landroid/view/View;->setContentDescription(Ljava/lang/CharSequence;)V
@@ -1853,11 +1879,11 @@
     :goto_12
     const/16 v1, 0x96
 
-    if-eqz v9, :cond_27
+    if-eqz v9, :cond_29
 
     iget-boolean v0, v0, Lz4/G;->f:Z
 
-    if-eqz v0, :cond_26
+    if-eqz v0, :cond_28
 
     new-instance v0, LU1/a;
 
@@ -1865,7 +1891,7 @@
 
     goto :goto_13
 
-    :cond_26
+    :cond_28
     new-instance v0, LU1/b;
 
     invoke-direct {v0, v4}, LU1/b;-><init>(Landroid/view/View;)V
@@ -1885,12 +1911,12 @@
 
     return-void
 
-    :cond_27
-    if-eqz v10, :cond_29
+    :cond_29
+    if-eqz v10, :cond_2b
 
     iget-boolean v2, v0, Lz4/G;->f:Z
 
-    if-eqz v2, :cond_28
+    if-eqz v2, :cond_2a
 
     new-instance v2, LU1/a;
 
@@ -1898,7 +1924,7 @@
 
     goto :goto_14
 
-    :cond_28
+    :cond_2a
     new-instance v2, LU1/b;
 
     invoke-direct {v2, v4}, LU1/b;-><init>(Landroid/view/View;)V
@@ -1922,25 +1948,25 @@
 
     return-void
 
-    :cond_29
+    :cond_2b
     invoke-static {v4}, LU1/a;->e(Landroid/view/View;)V
 
     return-void
 
-    :cond_2a
-    if-eqz v3, :cond_2b
+    :cond_2c
+    if-eqz v3, :cond_2d
 
     goto :goto_15
 
-    :cond_2b
+    :cond_2d
     move v5, v15
 
     :goto_15
-    if-eqz v9, :cond_2d
+    if-eqz v9, :cond_2f
 
     iget-boolean v0, v0, Lz4/G;->f:Z
 
-    if-eqz v0, :cond_2c
+    if-eqz v0, :cond_2e
 
     new-instance v0, LU1/c;
 
@@ -1948,7 +1974,7 @@
 
     goto :goto_16
 
-    :cond_2c
+    :cond_2e
     new-instance v0, LU1/d;
 
     invoke-direct {v0, v4}, LU1/d;-><init>(Landroid/view/View;)V
@@ -1964,12 +1990,12 @@
 
     return-void
 
-    :cond_2d
-    if-eqz v10, :cond_2f
+    :cond_2f
+    if-eqz v10, :cond_31
 
     iget-boolean v0, v0, Lz4/G;->f:Z
 
-    if-eqz v0, :cond_2e
+    if-eqz v0, :cond_30
 
     new-instance v0, LU1/c;
 
@@ -1977,7 +2003,7 @@
 
     goto :goto_17
 
-    :cond_2e
+    :cond_30
     new-instance v0, LU1/d;
 
     invoke-direct {v0, v4}, LU1/d;-><init>(Landroid/view/View;)V
@@ -1993,7 +2019,7 @@
 
     return-void
 
-    :cond_2f
+    :cond_31
     sget-object v0, Li0/E;->a:Ljava/util/WeakHashMap;
 
     invoke-virtual {v4, v15}, Landroid/view/View;->setAlpha(F)V
@@ -2008,6 +2034,8 @@
 
     return-void
 
+    nop
+
     :pswitch_data_0
     .packed-switch 0xc0
         :pswitch_12
